src/FakeItEasy/ExceptionThrowerConfigurationExtensions.cs

```diff
@@ -130,6 +134,11 @@ public static IAfterCallSpecifiedConfiguration Throws<T1>(this IExceptionThrower
                 });
         }
 
+#if DISABLED
+
+        // This method already has a generic parameter, and it can't be inferred; if we add another, we make it unusable,
+        // especially since the new one is something unintelligible like IAnyCallConfigurationWithReturnTypeSpecificied<Foo>...
+

```

Comment:
> And `RuleBuilder.ReturnValueConfiguration<TMember>`

So it does!

> I used the R# "Derived symbols" command (<kbd>Alt</kbd>+<kbd>End</kbd>)

So did I. And I **swear** that `RuleBuilder.ReturnValueConfiguration<TMember>` didn't show up. And now it does. User error, I'm sure.